tests/forms_tests/field_tests/test_imagefield.py

```diff
@@ -86,3 +86,14 @@ def test_widget_attrs_accept_false(self):
         f = ImageField(widget=FileInput(attrs={'accept': False}))
         self.assertEqual(f.widget_attrs(f.widget), {})
         self.assertWidgetRendersTo(f, '<input type="file" name="f" required id="id_f" />')
+
+    def test_unsupported_image_type(self):
+        f = ImageField()
+        img_path = get_img_path('filepath_test_files/unimplemented_dxgi_format.dds')
```

Comment:
Would using the Python test file for that to avoid adding a fixture work as well?

Reply:
Hi Simon - it works if I create a 'fake' image with `bytearray`. Have I interpreted your comment correctly?

Reply:
@smithdc1 it does thanks!